docker/python3-ubi/verify.py

```diff
@@ -0,0 +1,15 @@
+import warnings
```

Comment:
This file should be a symbolic link to the python3/verify.py file as we do in python3-deb. See for example: https://github.com/demisto/dockerfiles/blob/master/docker/python3-deb/verify.py

Reply:
done